feat: NIM 2.22.0 -- add configure-log-profiles (#1850)
nginx/documentation: content/nim/waf-integration/policies-and-logs/log-profiles/configure-log-profile.md

---
nd-content-type: how-to
nd-docs: DOCS-000
nd-product: NIMNGR
title: Configure and deploy log profiles
description: "Configure F5 WAF for NGINX security log profiles in NGINX Instance Manager, controlling request filtering, log format, size limits, and deployment to NGINX instances."
weight: 100
toc: true
nd-keywords: "log profiles, WAF, NGINX Instance Manager, NIM, security logs, app protect, request logs, traffic logs, Splunk, ArcSight, syslog"
nd-summary: >
  Configure log profiles for F5 WAF for NGINX security logs in NGINX Instance Manager.
  Log profiles define which HTTP requests are captured, how log messages are formatted, where logs are sent, and what security event details are included.
  Log profiles must be compiled into a bundle before deployment to NGINX instances.
nd-audience: operator
---

## Overview

> Use this guide to configure log profiles for F5 WAF for NGINX security logs in NGINX Instance Manager. Security logs (also called Request logs or Traffic logs) show what F5 WAF detects and how it processes traffic according to your policies. F5 WAF uses its own logging mechanism rather than NGINX's default access logging.

Reviewer: ```suggestion
Use this guide to configure log profiles for F5 WAF for NGINX security logs in F5 NGINX Instance Manager. Security logs (also called Request logs or Traffic logs) provide visibility into what F5 WAF for NGINX detects and how F5 WAF for NGINX processes traffic according to your policies. F5 WAF for NGINX uses its own logging mechanism rather than NGINX's default access logging.
```
